src/controllers/bets.ts

```diff
@@ -4,6 +4,11 @@ import { matchedData } from 'express-validator';
 import handleHttpError from '../utils/handleErrors';
 import models from '../models/index';
 
+/**
+ * Get a list of bets items from the database
+ * @param req
```

Comment:
I mean here, this comments, the other comments are just the result of the compilation